core/src/main/kotlin/com/justai/jaicf/model/activation/ActivationStrategy.kt

```diff
@@ -0,0 +1,26 @@
+package com.justai.jaicf.model.activation
+
+import com.justai.jaicf.activator.BaseActivator
+import com.justai.jaicf.context.BotContext
+
+/**
+ * Abstraction for strategy, using which activation will be selected from list.
+ * */
+interface ActivationStrategy {
+
+    /**
+     * Selects [Activation] from list using implemented strategy.
+     *
+     * @param botContext a current user's [BotContext]
+     * @param activations list of all available activations
+     * @return the most relevant [Activation] in terms of certain implementation of [BaseActivator]
```

Comment:
**Must be fixed**
As it's no longer related to an `Activator` implementation, let's rewrite this row.